iptables/init.sls

```diff
@@ -5,8 +5,16 @@
 {% from "iptables/map.jinja" import firewall with context %}
 {% set install = firewall.install %}
 {% set strict_mode = firewall.strict %}
+{% set ipv6 = firewall.get('ipv6', False) %}
 {% set global_block_nomatch = firewall.block_nomatch %}
 {% set packages = firewall.pkgs %}
+{% set ipv4 = 'IPv4' %}
+{% set ipv6 = 'IPv6' %}
+{% set protocols = [ipv4] %}
+{% if ipv6 %}
```

Comment:
This will always evaluate to `true` due to the reuse of the variable in [line 12](https://github.com/saltstack-formulas/iptables-formula/pull/32/files#diff-63596a5ee7fcee8f58ab4593d8745902R12)